apps/web/pages/api/app-store/installed.ts

```diff
@@ -25,6 +25,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         res.status(404);
       }
     } catch (error) {
+      console.log(error);
```

Comment:
Is there a better place to log it than console or is that planned to be attached at a later stage?